features-json/http3.json

```
@@ -444,19 +444,19 @@
       "13.2":"n",
       "13.3":"n",
       "13.4-13.7":"n",
-      "14.0-14.4":"n d",
-      "14.5-14.8":"n d",
-      "15.0-15.1":"n d",
-      "15.2-15.3":"n d",
-      "15.4":"n d",
-      "15.5":"n d",
-      "15.6":"n d",
-      "16.0":"n d",
-      "16.1":"n d",
-      "16.2":"n d",
-      "16.3":"n d",
-      "16.4":"n d",
-      "16.5":"n d"
+      "14.0-14.4":"a #4",
+      "14.5-14.8":"a #4",
+      "15.0-15.1":"a #4",
+      "15.2-15.3":"a #4",
+      "15.4":"a #4",
+      "15.5":"a #4",
+      "15.6":"a #4",
+      "16.0":"a #4",
+      "16.1":"a #4",
+      "16.2":"a #4",
+      "16.3":"a #4",
+      "16.4":"a #4",
+      "16.5":"a #4"
```

For `ios_saf` most likely note 5 was intended, right? :)
(Because iOS != macOS 0:-))

Like so:

```suggestion
      "14.0-14.4":"a #5",
      "14.5-14.8":"a #5",
      "15.0-15.1":"a #5",
      "15.2-15.3":"a #5",
      "15.4":"a #5",
      "15.5":"a #5",
      "15.6":"a #5",
      "16.0":"a #5",
      "16.1":"a #5",
      "16.2":"a #5",
      "16.3":"a #5",
      "16.4":"a #5",
      "16.5":"a #5"
```